addons/web/static/src/js/core/smooth_scroll_on_drag.js

```diff
@@ -55,7 +55,9 @@ const SmoothScrollOnDrag = Class.extend(mixins.ParentedMixin, {
      * @constructor
      * @param {Object} parent The parent widget that uses this class.
      * @param {jQuery} $element The element the smooth scroll on drag has to be set on.
-     * @param {jQuery} $scrollTarget The element the scroll will be triggered on.
+     * @param {jQuery} $target Container for drop element. $scrollTarget is this closest
+     *        scrollable parent element.
+     * @param {jQuery} $container The element the scroll will be triggered on.
```

Comment:
May I kindly benefit of your commit and ask you to change the type of the params options.scrollBoundaries.*?
I made a mistake in their types 😱 (it is have written Object although they are boolean)...
Thanks 😄

Reply:
Indeed, an error, this comment should not be present.